build/gen_stub.php

```diff
@@ -1864,6 +1868,28 @@ private function getParameterSection(DOMDocument $doc): DOMElement {
         return $parametersRefSec;
     }
 
+    /** @param array<string, FuncInfo> $aliasMap */
+    public function createDeprecationWarning(DOMDocument $doc, array $aliasMap): DOMElement {
+        /* Creation of <refsynopsisdiv> */
+        $refsynopsisDiv = $doc->createElement('refsynopsisdiv');
+        $deprecationEntity = $doc->createEntityReference(
+            'warn.deprecated.' . (isset($aliasMap[$this->name->__toString()]) ? 'alias' : 'function-') .
+            str_replace(".", "-", $this->deprecatedSince) . "-0"
+        );
+        $refsynopsisDiv->appendChild(new DOMText("\n  "));
```

Comment:
You can actually simplify DOMText insertions as follows:
```suggestion
        $refsynopsisDiv->append("\n  ");
```

or even combine the 3 appendChilds into one, but I guess you didn't do it because it would be a long line or would have to be multilined anyway.